gateway/gateway.go

```diff
@@ -1379,6 +1474,13 @@ func (gateway *HandleT) StartWebHandler(ctx context.Context) error {
 	srvMux.HandleFunc("/v1/webhook", gateway.webhookHandler.RequestHandler).Methods("POST", "GET")
 	srvMux.HandleFunc("/beacon/v1/batch", gateway.beaconBatchHandler).Methods("POST")
 
+	//Warehouse Async Jobs API
+	srvMux.HandleFunc("/v1/wh-jobs/add", gateway.warehouseJobAddHandler).Methods("POST")
+	srvMux.HandleFunc("/v1/wh-jobs/status", gateway.warehouseJobStatusHandler).Methods("POST")
```

Comment:
Why not mount them on `v1/warehouse/jobs`. Then we won't need add as POST would classify creation of job and on same note, `GET` endpoint could be `v1/warehouse/jobs/:id` right ?

Reply:
Lets take this up as another task.

Reply:
This is done.